src/Faker/Provider/Address.php

```diff
@@ -104,20 +104,24 @@ public static function country()
     }
 
     /**
-     * @example 77.147489
-     * @return float Uses signed degrees format (returns a float number between -90 and 90)
+     * @example '77.147489'
+     * @param int $min
+     * @param int $max
+     * @return string
      */
-    public static function latitude()
+    public static function latitude($min = -90, $max = 90)
     {
-        return static::randomFloat(6, 0, 180) - 90;
+        return number_format(mt_rand($min * 1000000, $max * 1000000)/1000000, 6);
```

Comment:
this is a coordinate, not a "just" float number. So I decided to update lat lng functions